pod.go

```diff
@@ -584,8 +595,8 @@ func (p *Pod) delete() error {
 		return err
 	}
 
-	if state.State != StateReady && state.State != StateStopped {
-		return fmt.Errorf("Pod not ready or stopped, impossible to delete")
+	if state.State != StateReady && state.State != StatePaused && state.State != StateStopped {
```

Comment:
I was imagining here that if you paused a pod for security purposes (strange activity inside it for example), you might want to immediately delete it without having to first resume it (since resuming would allow that rogue process to continue for a while before you deleted it).